test/integration/helpers/misc.ts

```diff
@@ -3,6 +3,7 @@
 import * as path from "path";
```

Comment:
What is your feeling on utility files like this?

On the one hand having a dumping ground is handy, on the other I am tempted to split out each key function into its own file named for the function.

How do you think we should deal with util files?

Reply:
I think I stopped the refactoring work halfway. I wanted to go from a single helper file to modules of helpers grouped by functionality. I can think of 3 approaches:
1 - move each function to a single file, without categorizing
2 - move each function a single file, categorizing them inside folders (i.e. `matchers/uriEqual.ts`, etc)
3 - have each category be a file with multiple exports (current approach for e.g. `helpers/files.ts`).

Which one do you like more ? I favor numbers 2 and 3 but the downside is they require thinking of some categorization/grouping and we may be tempted at some point to have an uncategorized/dump file.

Reply:
The honest answer is, I prefer 2 but usually get to 2 via 3. I don't want to be committing to an abstraction too early (so dump in a broad category file - `helper.js` for the win), but if it clearly useful and widely used I start feeling guilty and want to move it out to its own file.

So like a coward, I am going to defer to your judgement on this one, either 2 or 3 in this case.